spec/dor_indexing/indexers/descriptive_metadata_indexer_title_spec.rb

```diff
@@ -5,731 +5,448 @@
 
   let(:bare_druid) { 'qy781dy0220' }
   let(:druid) { "druid:#{bare_druid}" }
+  let(:titles) { [{ value: 'ignore me' }] }
   let(:doc) { indexer.to_solr }
-  let(:cocina) do
-    build(:dro, id: druid).new(
-      description: description.merge(purl: "https://purl.stanford.edu/#{bare_druid}")
-    )
+  let(:description) do
+    {
+      title: titles,
+      purl: "https://purl.stanford.edu/#{bare_druid}"
+    }
   end
+  let(:cocina) { build(:dro, id: druid).new(description:) }
 
-  describe 'title mappings from Cocina to Solr sw_display_title_tesim' do
-    describe 'single untyped title' do
-      # Select value; status: primary may or may not be present
-      let(:description) do
-        {
-          title: [
-            {
-              value: 'Title'
-            }
-          ]
-        }
+  describe 'title mappings from Cocina to Solr' do
```

Comment:
while the changes in this file look daunting, they can be explained:
1.  remove all the specs that are now in cocina-models (which does the complicated parsing)
2.  have specs here to capture some of the nuances of title parsing and recombining, and the differences between the different flavors of title fields.